contracts/test/bridge/tokenBridge.ts

```diff
@@ -0,0 +1,184 @@
+import { TIMEOUT } from '../constants'
```

Comment:
These tests should not go under our `/contracts` project, but we have a specific integration project for it already: https://github.com/smartcontractkit/chainlink-starknet/tree/develop/packages-ts/integration-starkgate

Let's move it all there.